resources/assets/less/store/slider.less

```diff
@@ -0,0 +1,59 @@
+/*!
+ * jQuery UI Slider 1.8.23
+ *
+ * Copyright 2012, AUTHORS.txt (http://jqueryui.com/about)
+ * Dual licensed under the MIT or GPL Version 2 licenses.
+ * http://jquery.org/license
+ *
+ * http://docs.jquery.com/UI/Slider#theming
+ */
+
+.ui-corner-all {
```

Comment:
more like global override than just store. Or just make it part of the store slider bem thingy if any.